torchgeo/trainers/regression.py

```diff
@@ -97,7 +97,21 @@ def configure_losses(self) -> None:
             )
 
     def configure_metrics(self) -> None:
-        """Initialize the performance metrics."""
+        """Initialize the performance metrics.
+
+        * Root MeanSquared Error: The squared root of the mean squared error. Lower
```

Comment:
could be clearer: The square root of the average of the squared differences between the predicted and actual values